### ISSUE TITLE
Keyboard closes after changing focus from another Text Field.

### ISSUE THREAD

**Issue by horeyes** [vcooley/nativescript-input-mask#5]
Hello,

I'm using this plugin and every time I change focus from another text field the keyboard closes for some reason. Below I provide a sample of the issue.

![question](https://user-images.githubusercontent.com/12671900/56066250-3253f380-5d45-11e9-85bb-580bf78d5464.gif)

To recreate the behaviour just add Text Fields on top of the input-mask text field. But here's the sample app I used to create the gif. 
[TextViewForm.zip](https://github.com/vcooley/nativescript-input-mask/files/3075180/TextViewForm.zip)

Thanks in advanced.

**Comment by mcanikhilprajapati** [comment]
@horeyes Same issue I have. do you find any solution?

**Comment by horeyes** [comment]
No, I haven't found a solution yet. I discovered that the issue happens whenever you apply the mask to the plugin. If you remove the mask property then you don't get the issue. I know this is not a solution, but it might help to figure out where's the issue happening.

**Comment by vcooley** [comment]
Thanks @horeyes for the report and the additional info about the mask property. I'll take a look at it soon.

**Comment by mcanikhilprajapati** [comment]
thanks for the update @vcooley  @horeyes
